File: web/src/app/model/city.ts
export class City {
    id?: number;
    name?: string;
    country?: string;
    countryCode?: string;

    constructor() {
        if (this.id === undefined) this.id = null;
        if (this.name === undefined) this.name = null;
        if (this.country === undefined) this.country = null;
        if (this.countryCode === undefined) this.countryCode = null;
    }
}

File: api/src/main/java/com/timoteo/jobsearch/model/City.java
package com.timoteo.jobsearch.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
@JsonIgnoreProperties(ignoreUnknown = true)
public class City {
    private Long id;
    private String name;
    private String country;
    private String countryCode;
}
